src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/CollectionModulesConsistency.fs

```diff
@@ -8,11 +8,17 @@ open NUnit.Framework
 open FsCheck
 open Utils
 
+/// helper function that creates labeled FsCheck properties for equality comparisons
+let consistency name sqs ls arr =
```

Comment:
by default it does not, if run with `Check.QuickThrowOnFailure` that method will throw the exception when the `Property` returns a failing case.